src/u2f.c

```diff
@@ -865,35 +824,20 @@ static void _cmd_authenticate(const Packet* in_packet, Packet* out_packet)
     }
 }
 
-/**
- * Abort an existing authentication request.
- */
-static void _abort_authenticate(void)
-{
-    switch (_state.auth) {
-    case U2F_AUTHENTICATE_UNLOCKING:
-        rust_workflow_abort_current();
-        _clear_state();
-        break;
-    case U2F_AUTHENTICATE_CONFIRMING:
-        u2f_app_confirm_abort();
-        _clear_state();
-        break;
-    case U2F_AUTHENTICATE_WAIT_REFRESH:
-        _stop_refresh_webpage_screen();
-        _clear_state();
-        break;
-    default:
-        Abort("Bad U2F register abort status");
-    }
-}
-
 /**
  * Process a U2F message
  */
 static void _cmd_msg(const Packet* in_packet, Packet* out_packet, const size_t max_out_len)
 {
     (void)max_out_len;
+
+    // Give the user 5s to respond to u2f requests (the FIDO Client in u2f is not required to ping
+    // every 200ms like the BitBoxApp does even though most implementations do).
+    usb_processing_timeout_reset(-50);
```

Comment:
If it times out the confirm screen just disappears? 

Do you know of a browser that does not ping to keep it alive?

Reply:
Isn't 5s seems a bit too short. I'd bump it to at least 10s, maybe 15s. Would be frustrating to miss it. Do you know what timeout Yubikey uses?

Reply:
So we can probably remove this timeout, even if 5s is what I found in other open source implementations.

(btw it wouldn't actually mean that the user would only have 5s. because I guess all FIDO clients try again within that 5s.) I think FIDO clients typically retry up to at least 30s. maybe longer